docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -393,13 +332,13 @@ Follow these instructions to start working on your Python control code:
 
    {{% snippet "show-secret.md" %}}
 
-4. Paste this code sample into a new file in the `plant-watering-robot` directory you created on your Pi.
+4. Paste this code sample into a new file in the virtual directory you created on your Pi.
```

Comment:
what's this directory supposed to be? previously it was a directory you instructed a user to create

Reply:
the virtual environment. since they are not testing they don't need a specific directory, they will only have one script

Reply:
then you can't call that a virtual directory

```suggestion
4. Paste this code sample into a new file on your Pi.
```